cmd/reposervercontroller/main.go

```diff
@@ -38,7 +38,6 @@ import (
 	"github.com/kanisterio/kanister/pkg/log"
 	"github.com/kanisterio/kanister/pkg/resource"
 	"github.com/kanisterio/kanister/pkg/validatingwebhook"
-	//+kubebuilder:scaffold:imports
```

Comment:
Its a marker, not import. I tried to find what exactly it does, bit couldn't find much info.
Is it safe to remove this?

Reply:
I think it got removed automatically. Will re-add it until we know for sure it's not needed.

Reply:
Added it back. This [issue](https://github.com/kubernetes-sigs/kubebuilder/issues/1487#issuecomment-823864494) explains the reason of this. We can focus on this [part](https://github.com/kubernetes-sigs/kubebuilder/issues/1487#issuecomment-1485346413) of it.